controller/src/beerocks/master/tasks/dynamic_channel_selection_task.h

```diff
@@ -20,6 +20,7 @@
 // Helper MACROs for Enum/String generation
 #define GENERATE_ENUM(ENUM) ENUM,
 #define GENERATE_STRING(STRING) #STRING,
+#define NO_SCAN_IN_PROGRESS -1
```

Comment:
If you change `m_scanning` to `m_scanning_task_id`, no need for this define.

Reply:
changed `m_scanning` to `m_scanning_task_id`